portfolio/src/main/java/com/google/sps/servlets/QuoteGenerator.java

```diff
@@ -9,10 +12,38 @@
 /** Handles requests sent to the /quote URL endpoint */
 @WebServlet("/quote")
 public class QuoteGenerator extends HttpServlet {
+  /**
+   * Build the quotes using the author name and quote text.
+   *
+   * @param authorName The author of the quote's name
```

Comment:
nit: Chanhe comment to "Name of the quote author" ?

Reply:
Corrected the grammatical error